xgi/algorithms/centrality.py

```diff
@@ -392,3 +387,161 @@ def katz_centrality(H, cutoff=100):
         c *= 1 / norm(c, 1)
     nodedict = dict(zip(range(n), H.nodes))
     return {nodedict[idx]: c[idx] for idx in nodedict}
+
+
+def h_eigenvector_tensor_centrality(H, max_iter=100, tol=1e-6):
+    """Compute the H-eigenvector centrality of a hypergraph.
```

Comment:
maybe add one line of additional info saying for example what H stands for, or what the specificity of this centrality is?

Reply:
I'm thinking of adding a reference for H- (and Z-) eigenvectors of tensors.

Reply:
Done.